leakcanary-android/src/main/java/com/squareup/leakcanary/AndroidExcludedRefs.java

```diff
@@ -212,7 +212,7 @@
     }
   },
 
-  ACCOUNT_MANAGER(SDK_INT <= LOLLIPOP_MR1) {
+  ACCOUNT_MANAGER(SDK_INT <= O_MR1) {
```

Comment:
Why not SDK_INT > ECLAIR ?  This bug will be not fixed by Google so why update the latest version as maximum ?

Reply:
@JakeWharton @jrodbx  Hi guys. Can someone pickup this issue ? Is really annoying to still have this issue. Please answer in case that can be solved without excluding this memory leak.

Reply:
This is a bug in AOSP, involving IPC. Google can fix it if they decide to do so.